### ISSUE TITLE
Course load should consider multiple dimensions

### ISSUE THREAD

**Issue by ppannuto** [gsingh93/class-load#3]
Currently it's just an aggregate "difficulty", which is good on the surface, but I think a lot of the value can come from understanding interactions. E.g. one of the most common pieces of advice is "that's too many programming-heavy courses", finding a few good metrics to capture some of the deeper interactions would make this a more powerful tool.

**Comment by gsingh93** [comment]
True, but I'm not sure what other metrics should be considered. I also can't think of a way I'd display this information to the user (I'm not a front-end dev).
